rust/src/named_graph.rs

```diff
@@ -23,6 +23,16 @@ predicates! {
         }
     }
 
+    pub semidet fn delete_named_graph(context, store_term, graph_name_term) {
+        let store: WrappedStore = store_term.get_ex()?;
+        let graph_name: PrologText = graph_name_term.get_ex()?;
+
+        match context.try_or_die(store.open(&graph_name))? {
```

Comment:
We could implemented `store.delete(&graph_name)` directly, which would make this code a bit simpler. but it's not strictly necessary.